FIXME Do not build in all drivers in multi_defconfig!
---
 arch/m68k/configs/amiga_defconfig    |    4 -
 arch/m68k/configs/apollo_defconfig   |    4 -
 arch/m68k/configs/atari_defconfig    |    8 ++-
 arch/m68k/configs/bvme6000_defconfig |    2 
 arch/m68k/configs/hp300_defconfig    |    4 -
 arch/m68k/configs/mac_defconfig      |    4 -
 arch/m68k/configs/multi_defconfig    |   81 ++++++++++++++++++-----------------
 arch/m68k/configs/mvme147_defconfig  |    2 
 arch/m68k/configs/mvme16x_defconfig  |    2 
 arch/m68k/configs/q40_defconfig      |    4 -
 arch/m68k/configs/sun3_defconfig     |    4 -
 arch/m68k/configs/sun3x_defconfig    |    4 -
 12 files changed, 65 insertions(+), 58 deletions(-)

--- a/arch/m68k/configs/amiga_defconfig
+++ b/arch/m68k/configs/amiga_defconfig
@@ -1,7 +1,7 @@
 #
 # Automatically generated make config: don't edit
 # Linux kernel version: 2.6.25
-# Wed Apr 23 22:51:57 2008
+# Wed Apr 23 09:28:30 2008
 #
 CONFIG_M68K=y
 CONFIG_MMU=y
@@ -793,7 +793,7 @@ CONFIG_FB_CFB_IMAGEBLIT=y
 # CONFIG_FB_SYS_COPYAREA is not set
 # CONFIG_FB_SYS_IMAGEBLIT is not set
 # CONFIG_FB_SYS_FOPS is not set
-CONFIG_FB_DEFERRED_IO=y
+# CONFIG_FB_DEFERRED_IO is not set
 # CONFIG_FB_SVGALIB is not set
 # CONFIG_FB_MACMODES is not set
 # CONFIG_FB_BACKLIGHT is not set
--- a/arch/m68k/configs/apollo_defconfig
+++ b/arch/m68k/configs/apollo_defconfig
@@ -1,7 +1,7 @@
 #
 # Automatically generated make config: don't edit
 # Linux kernel version: 2.6.25
-# Wed Apr 23 22:52:44 2008
+# Wed Apr 23 09:28:34 2008
 #
 CONFIG_M68K=y
 CONFIG_MMU=y
@@ -694,7 +694,7 @@ CONFIG_FB_CFB_IMAGEBLIT=y
 # CONFIG_FB_SYS_COPYAREA is not set
 # CONFIG_FB_SYS_IMAGEBLIT is not set
 # CONFIG_FB_SYS_FOPS is not set
-CONFIG_FB_DEFERRED_IO=y
+# CONFIG_FB_DEFERRED_IO is not set
 # CONFIG_FB_SVGALIB is not set
 # CONFIG_FB_MACMODES is not set
 # CONFIG_FB_BACKLIGHT is not set
--- a/arch/m68k/configs/atari_defconfig
+++ b/arch/m68k/configs/atari_defconfig
@@ -1,7 +1,7 @@
 #
 # Automatically generated make config: don't edit
 # Linux kernel version: 2.6.25
-# Wed Apr 23 22:47:46 2008
+# Wed Apr 23 09:28:36 2008
 #
 CONFIG_M68K=y
 CONFIG_MMU=y
@@ -113,12 +113,15 @@ CONFIG_CLASSIC_RCU=y
 # CONFIG_SUN3 is not set
 # CONFIG_AMIGA is not set
 CONFIG_ATARI=y
+CONFIG_ATARI_ROM_ISA=y
 # CONFIG_MAC is not set
 # CONFIG_APOLLO is not set
 # CONFIG_VME is not set
 # CONFIG_HP300 is not set
 # CONFIG_SUN3X is not set
 # CONFIG_Q40 is not set
+CONFIG_NATFEAT=y
+CONFIG_NFETH=y
 
 #
 # Processor type
@@ -560,6 +563,7 @@ CONFIG_VETH=m
 CONFIG_NET_ETHERNET=y
 CONFIG_MII=m
 CONFIG_ATARILANCE=m
+CONFIG_ATARI_ETHERNEC=m
 # CONFIG_IBM_NEW_EMAC_ZMII is not set
 # CONFIG_IBM_NEW_EMAC_RGMII is not set
 # CONFIG_IBM_NEW_EMAC_TAH is not set
@@ -737,7 +741,7 @@ CONFIG_FB_CFB_IMAGEBLIT=y
 # CONFIG_FB_SYS_COPYAREA is not set
 # CONFIG_FB_SYS_IMAGEBLIT is not set
 # CONFIG_FB_SYS_FOPS is not set
-CONFIG_FB_DEFERRED_IO=y
+# CONFIG_FB_DEFERRED_IO is not set
 # CONFIG_FB_SVGALIB is not set
 # CONFIG_FB_MACMODES is not set
 # CONFIG_FB_BACKLIGHT is not set
--- a/arch/m68k/configs/bvme6000_defconfig
+++ b/arch/m68k/configs/bvme6000_defconfig
@@ -1,7 +1,7 @@
 #
 # Automatically generated make config: don't edit
 # Linux kernel version: 2.6.25
-# Wed Apr 23 22:53:06 2008
+# Wed Apr 23 09:28:39 2008
 #
 CONFIG_M68K=y
 CONFIG_MMU=y
--- a/arch/m68k/configs/hp300_defconfig
+++ b/arch/m68k/configs/hp300_defconfig
@@ -1,7 +1,7 @@
 #
 # Automatically generated make config: don't edit
 # Linux kernel version: 2.6.25
-# Wed Apr 23 22:53:36 2008
+# Wed Apr 23 09:28:41 2008
 #
 CONFIG_M68K=y
 CONFIG_MMU=y
@@ -703,7 +703,7 @@ CONFIG_FB_CFB_IMAGEBLIT=y
 # CONFIG_FB_SYS_COPYAREA is not set
 # CONFIG_FB_SYS_IMAGEBLIT is not set
 # CONFIG_FB_SYS_FOPS is not set
-CONFIG_FB_DEFERRED_IO=y
+# CONFIG_FB_DEFERRED_IO is not set
 # CONFIG_FB_SVGALIB is not set
 # CONFIG_FB_MACMODES is not set
 # CONFIG_FB_BACKLIGHT is not set
--- a/arch/m68k/configs/mac_defconfig
+++ b/arch/m68k/configs/mac_defconfig
@@ -1,7 +1,7 @@
 #
 # Automatically generated make config: don't edit
 # Linux kernel version: 2.6.25
-# Wed Apr 23 22:54:06 2008
+# Wed Apr 23 09:28:44 2008
 #
 CONFIG_M68K=y
 CONFIG_MMU=y
@@ -736,7 +736,7 @@ CONFIG_FB_CFB_IMAGEBLIT=y
 # CONFIG_FB_SYS_COPYAREA is not set
 # CONFIG_FB_SYS_IMAGEBLIT is not set
 # CONFIG_FB_SYS_FOPS is not set
-CONFIG_FB_DEFERRED_IO=y
+# CONFIG_FB_DEFERRED_IO is not set
 # CONFIG_FB_SVGALIB is not set
 CONFIG_FB_MACMODES=y
 # CONFIG_FB_BACKLIGHT is not set
--- a/arch/m68k/configs/multi_defconfig
+++ b/arch/m68k/configs/multi_defconfig
@@ -1,7 +1,7 @@
 #
 # Automatically generated make config: don't edit
 # Linux kernel version: 2.6.25
-# Wed Apr 23 22:39:35 2008
+# Wed Apr 23 22:43:51 2008
 #
 CONFIG_M68K=y
 CONFIG_MMU=y
@@ -24,7 +24,7 @@ CONFIG_DEFCONFIG_LIST="/lib/modules/$UNA
 CONFIG_EXPERIMENTAL=y
 CONFIG_BROKEN_ON_SMP=y
 CONFIG_INIT_ENV_ARG_LIMIT=32
-CONFIG_LOCALVERSION="-multi"
+CONFIG_LOCALVERSION="multi"
 CONFIG_LOCALVERSION_AUTO=y
 CONFIG_SWAP=y
 CONFIG_SYSVIPC=y
@@ -113,6 +113,7 @@ CONFIG_CLASSIC_RCU=y
 # CONFIG_SUN3 is not set
 CONFIG_AMIGA=y
 CONFIG_ATARI=y
+CONFIG_ATARI_ROM_ISA=y
 CONFIG_MAC=y
 CONFIG_NUBUS=y
 CONFIG_M68K_L2_CACHE=y
@@ -125,6 +126,8 @@ CONFIG_HP300=y
 CONFIG_DIO=y
 CONFIG_SUN3X=y
 CONFIG_Q40=y
+CONFIG_NATFEAT=y
+CONFIG_NFETH=y
 
 #
 # Processor type
@@ -446,11 +449,11 @@ CONFIG_FW_LOADER=m
 # CONFIG_SYS_HYPERVISOR is not set
 CONFIG_CONNECTOR=m
 # CONFIG_MTD is not set
-CONFIG_PARPORT=m
+CONFIG_PARPORT=y
 # CONFIG_PARPORT_PC is not set
-CONFIG_PARPORT_AMIGA=m
-CONFIG_PARPORT_MFC3=m
-CONFIG_PARPORT_ATARI=m
+CONFIG_PARPORT_AMIGA=y
+CONFIG_PARPORT_MFC3=y
+CONFIG_PARPORT_ATARI=y
 # CONFIG_PARPORT_GSC is not set
 # CONFIG_PARPORT_AX88796 is not set
 CONFIG_PARPORT_1284=y
@@ -621,21 +624,22 @@ CONFIG_VETH=m
 # CONFIG_ARCNET is not set
 # CONFIG_PHYLIB is not set
 CONFIG_NET_ETHERNET=y
-CONFIG_MII=m
-CONFIG_ARIADNE=m
-CONFIG_A2065=m
-CONFIG_HYDRA=m
-CONFIG_ZORRO8390=m
-CONFIG_APNE=m
+CONFIG_MII=y
+CONFIG_ARIADNE=y
+CONFIG_A2065=y
+CONFIG_HYDRA=y
+CONFIG_ZORRO8390=y
+CONFIG_APNE=y
 CONFIG_APOLLO_ELPLUS=y
 CONFIG_MAC8390=y
-CONFIG_MAC89x0=m
-CONFIG_MACSONIC=m
+CONFIG_MAC89x0=y
+CONFIG_MACSONIC=y
 CONFIG_MACMACE=y
 CONFIG_MVME147_NET=y
 CONFIG_MVME16x_NET=y
 CONFIG_BVME6000_NET=y
-CONFIG_ATARILANCE=m
+CONFIG_ATARILANCE=y
+CONFIG_ATARI_ETHERNEC=y
 CONFIG_SUN3LANCE=y
 CONFIG_HPLANCE=y
 # CONFIG_NET_VENDOR_3COM is not set
@@ -729,14 +733,14 @@ CONFIG_MOUSE_PS2_SYNAPTICS=y
 CONFIG_MOUSE_PS2_LIFEBOOK=y
 CONFIG_MOUSE_PS2_TRACKPOINT=y
 # CONFIG_MOUSE_PS2_TOUCHKIT is not set
-CONFIG_MOUSE_SERIAL=m
+CONFIG_MOUSE_SERIAL=y
 # CONFIG_MOUSE_INPORT is not set
 # CONFIG_MOUSE_LOGIBM is not set
 # CONFIG_MOUSE_PC110PAD is not set
-CONFIG_MOUSE_AMIGA=m
-CONFIG_MOUSE_ATARI=m
+CONFIG_MOUSE_AMIGA=y
+CONFIG_MOUSE_ATARI=y
 # CONFIG_MOUSE_VSXXXAA is not set
-CONFIG_MOUSE_HIL=m
+CONFIG_MOUSE_HIL=y
 CONFIG_INPUT_JOYSTICK=y
 # CONFIG_JOYSTICK_ANALOG is not set
 # CONFIG_JOYSTICK_A3D is not set
@@ -759,21 +763,21 @@ CONFIG_INPUT_JOYSTICK=y
 # CONFIG_JOYSTICK_DB9 is not set
 # CONFIG_JOYSTICK_GAMECON is not set
 # CONFIG_JOYSTICK_TURBOGRAFX is not set
-CONFIG_JOYSTICK_AMIGA=m
+CONFIG_JOYSTICK_AMIGA=y
 # CONFIG_JOYSTICK_JOYDUMP is not set
 # CONFIG_INPUT_TABLET is not set
 # CONFIG_INPUT_TOUCHSCREEN is not set
 CONFIG_INPUT_MISC=y
-CONFIG_INPUT_M68K_BEEP=m
+CONFIG_INPUT_M68K_BEEP=y
 # CONFIG_INPUT_UINPUT is not set
-CONFIG_HP_SDC_RTC=m
+CONFIG_HP_SDC_RTC=y
 
 #
 # Hardware I/O ports
 #
 CONFIG_SERIO=y
 # CONFIG_SERIO_SERPORT is not set
-CONFIG_SERIO_Q40KBD=m
+CONFIG_SERIO_Q40KBD=y
 # CONFIG_SERIO_PARKBD is not set
 CONFIG_HP_SDC=y
 CONFIG_HIL_MLC=y
@@ -789,7 +793,7 @@ CONFIG_VT_CONSOLE=y
 CONFIG_HW_CONSOLE=y
 CONFIG_VT_HW_CONSOLE_BINDING=y
 # CONFIG_SERIAL_NONSTANDARD is not set
-CONFIG_A2232=m
+CONFIG_A2232=y
 
 #
 # Serial drivers
@@ -802,13 +806,13 @@ CONFIG_A2232=m
 CONFIG_UNIX98_PTYS=y
 CONFIG_LEGACY_PTYS=y
 CONFIG_LEGACY_PTY_COUNT=256
-CONFIG_PRINTER=m
+CONFIG_PRINTER=y
 # CONFIG_LP_CONSOLE is not set
 # CONFIG_PPDEV is not set
 # CONFIG_IPMI_HANDLER is not set
 # CONFIG_HW_RANDOM is not set
 CONFIG_NVRAM=y
-CONFIG_GEN_RTC=m
+CONFIG_GEN_RTC=y
 CONFIG_GEN_RTC_X=y
 # CONFIG_DTLK is not set
 # CONFIG_R3964 is not set
@@ -862,7 +866,6 @@ CONFIG_FB_CFB_IMAGEBLIT=y
 # CONFIG_FB_SYS_COPYAREA is not set
 # CONFIG_FB_SYS_IMAGEBLIT is not set
 # CONFIG_FB_SYS_FOPS is not set
-CONFIG_FB_DEFERRED_IO=y
 # CONFIG_FB_SVGALIB is not set
 CONFIG_FB_MACMODES=y
 # CONFIG_FB_BACKLIGHT is not set
@@ -916,11 +919,11 @@ CONFIG_LOGO_MAC_CLUT224=y
 #
 # Sound
 #
-CONFIG_SOUND=m
-CONFIG_DMASOUND_ATARI=m
-CONFIG_DMASOUND_PAULA=m
-CONFIG_DMASOUND_Q40=m
-CONFIG_DMASOUND=m
+CONFIG_SOUND=y
+CONFIG_DMASOUND_ATARI=y
+CONFIG_DMASOUND_PAULA=y
+CONFIG_DMASOUND_Q40=y
+CONFIG_DMASOUND=y
 CONFIG_HID_SUPPORT=y
 CONFIG_HID=m
 # CONFIG_HID_DEBUG is not set
@@ -936,13 +939,13 @@ CONFIG_HIDRAW=y
 #
 # Character devices
 #
-CONFIG_ATARI_MFPSER=m
+CONFIG_ATARI_MFPSER=y
 CONFIG_ATARI_SCC=y
 CONFIG_ATARI_SCC_DMA=y
-CONFIG_ATARI_MIDI=m
-CONFIG_ATARI_DSP56K=m
+CONFIG_ATARI_MIDI=y
+CONFIG_ATARI_DSP56K=y
 CONFIG_AMIGA_BUILTIN_SERIAL=y
-CONFIG_MULTIFACE_III_TTY=m
+CONFIG_MULTIFACE_III_TTY=y
 CONFIG_MAC_SCC=y
 CONFIG_MAC_HID=y
 CONFIG_MVME147_SCC=y
@@ -977,8 +980,8 @@ CONFIG_XFS_FS=m
 # CONFIG_XFS_POSIX_ACL is not set
 # CONFIG_XFS_RT is not set
 CONFIG_OCFS2_FS=m
-CONFIG_OCFS2_FS_O2CB=m
-CONFIG_OCFS2_FS_USERSPACE_CLUSTER=m
+# CONFIG_OCFS2_FS_O2CB is not set
+# CONFIG_OCFS2_FS_USERSPACE_CLUSTER is not set
 # CONFIG_OCFS2_DEBUG_MASKLOG is not set
 # CONFIG_OCFS2_DEBUG_FS is not set
 CONFIG_DNOTIFY=y
@@ -1180,7 +1183,7 @@ CONFIG_CRYPTO_SEQIV=m
 #
 CONFIG_CRYPTO_CBC=y
 CONFIG_CRYPTO_CTR=m
-# CONFIG_CRYPTO_CTS is not set
+CONFIG_CRYPTO_CTS=m
 CONFIG_CRYPTO_ECB=m
 CONFIG_CRYPTO_LRW=m
 CONFIG_CRYPTO_PCBC=m
--- a/arch/m68k/configs/mvme147_defconfig
+++ b/arch/m68k/configs/mvme147_defconfig
@@ -1,7 +1,7 @@
 #
 # Automatically generated make config: don't edit
 # Linux kernel version: 2.6.25
-# Wed Apr 23 22:54:29 2008
+# Wed Apr 23 09:28:50 2008
 #
 CONFIG_M68K=y
 CONFIG_MMU=y
--- a/arch/m68k/configs/mvme16x_defconfig
+++ b/arch/m68k/configs/mvme16x_defconfig
@@ -1,7 +1,7 @@
 #
 # Automatically generated make config: don't edit
 # Linux kernel version: 2.6.25
-# Wed Apr 23 22:54:50 2008
+# Wed Apr 23 09:28:52 2008
 #
 CONFIG_M68K=y
 CONFIG_MMU=y
--- a/arch/m68k/configs/q40_defconfig
+++ b/arch/m68k/configs/q40_defconfig
@@ -1,7 +1,7 @@
 #
 # Automatically generated make config: don't edit
 # Linux kernel version: 2.6.25
-# Wed Apr 23 22:55:49 2008
+# Wed Apr 23 09:28:54 2008
 #
 CONFIG_M68K=y
 CONFIG_MMU=y
@@ -748,7 +748,7 @@ CONFIG_FB_CFB_IMAGEBLIT=y
 # CONFIG_FB_SYS_COPYAREA is not set
 # CONFIG_FB_SYS_IMAGEBLIT is not set
 # CONFIG_FB_SYS_FOPS is not set
-CONFIG_FB_DEFERRED_IO=y
+# CONFIG_FB_DEFERRED_IO is not set
 # CONFIG_FB_SVGALIB is not set
 # CONFIG_FB_MACMODES is not set
 # CONFIG_FB_BACKLIGHT is not set
--- a/arch/m68k/configs/sun3_defconfig
+++ b/arch/m68k/configs/sun3_defconfig
@@ -1,7 +1,7 @@
 #
 # Automatically generated make config: don't edit
 # Linux kernel version: 2.6.25
-# Wed Apr 23 22:56:22 2008
+# Wed Apr 23 09:28:57 2008
 #
 CONFIG_M68K=y
 CONFIG_MMU=y
@@ -684,7 +684,7 @@ CONFIG_FB=y
 # CONFIG_FB_SYS_COPYAREA is not set
 # CONFIG_FB_SYS_IMAGEBLIT is not set
 # CONFIG_FB_SYS_FOPS is not set
-CONFIG_FB_DEFERRED_IO=y
+# CONFIG_FB_DEFERRED_IO is not set
 # CONFIG_FB_SVGALIB is not set
 # CONFIG_FB_MACMODES is not set
 # CONFIG_FB_BACKLIGHT is not set
--- a/arch/m68k/configs/sun3x_defconfig
+++ b/arch/m68k/configs/sun3x_defconfig
@@ -1,7 +1,7 @@
 #
 # Automatically generated make config: don't edit
 # Linux kernel version: 2.6.25
-# Wed Apr 23 22:56:41 2008
+# Wed Apr 23 09:28:59 2008
 #
 CONFIG_M68K=y
 CONFIG_MMU=y
@@ -694,7 +694,7 @@ CONFIG_FB=y
 # CONFIG_FB_SYS_COPYAREA is not set
 # CONFIG_FB_SYS_IMAGEBLIT is not set
 # CONFIG_FB_SYS_FOPS is not set
-CONFIG_FB_DEFERRED_IO=y
+# CONFIG_FB_DEFERRED_IO is not set
 # CONFIG_FB_SVGALIB is not set
 # CONFIG_FB_MACMODES is not set
 # CONFIG_FB_BACKLIGHT is not set
